LiskHQ/lisk-desktop
# Prevent auto update of generator table

### What was the problem?

This PR resolves #5399

### How was it solved?

Replacing refetchInterval on generator table with ui button to enable refreshes of this table.

### How was it tested?

1. Go to http://localhost:8080/#/validators
2. Sort by commission
3. Wait 20 sec
4. Expected: List should not have been update
5. Click Refresh
6. Expected: List is now updated
